flatland_server/src/layer.cpp

```diff
@@ -301,7 +301,11 @@ void Layer::LoadFromBitmap(const cv::Mat &bitmap, double occupied_thresh,
 }
 
 void Layer::DebugVisualize() const {
-  ROS_WARN("======== 3d visualize? ===========");
+  // Don't try to visualized uninitalized layers
+  if (viz_name_.length() == 0) {
+    return;
```

Comment:
Not sure when will this happen, but do we need a warning here?

Reply:
That was an old debugging statement. I removed it.